src/aliased_buffer.h

```diff
@@ -9,6 +9,18 @@
 
 namespace node {
 
+template <class NativeT,
+          class V8T,
+          // SFINAE NativeT to be scalar
+          typename Trait = std::enable_if_t<std::is_scalar<NativeT>::value>>
+class OwningAliasedBufferBase;
+
+template <class NativeT,
+          class V8T,
+          // SFINAE NativeT to be scalar
+          typename Trait = std::enable_if_t<std::is_scalar<NativeT>::value>>
+class AliasedBufferViewBase;
```

Comment:
Either some code comments explaining the differences and why one would be used vs. the other or some information added to `src/README.md` explaining the same (or both) would be ideal